src/lightspeed_evaluation/core/constants.py

```diff
@@ -1,6 +1,6 @@
 """Common constants for evaluation framework."""
 
-DEFAULT_API_BASE = "http://localhost:8080"
+DEFAULT_API_BASE = "http://localhost:8080v1"
```

Comment:
Missing slash before and after v1

```suggestion
DEFAULT_API_BASE = "http://localhost:8080/v1/"
```

Reply:
updated